index.html

```diff
@@ -157,10 +151,7 @@ <h2 class="project-subtitle">RPM Governor</h2>
           <h2 class="project-subtitle">RPM Governor</h2>
```

Comment:
- [ ] Great job following the project design and your design is excellent :muscle: , however, I kindly suggest that you add one more project to your second row as it is on the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) design template given:
- [ ] I kindly suggest you reduce your project description to two to three lines of text as it is in the Figma design template.
- [ ] I kindly suggest that you add functionality such that the see project button is only visible when the user hovers over the description with a mouse, with a dark opacity disappearing like in the Figma design.

**Figma Design**
![image](https://user-images.githubusercontent.com/52934128/177666107-10db9688-b717-49c3-9593-affece58ddbe.png)
**Your Design**
![image](https://user-images.githubusercontent.com/52934128/177666205-cea4b8b3-9d74-4e62-ab25-6e3eeba3a05f.png)

Reply:
Yeah it is true ok i'm working 
thank you so much!